conf/evolutions/default/103.sql

```diff
@@ -0,0 +1,34 @@
+# --- !Ups
+DROP TABLE projects_deleted;
```

Comment:
We might want to keep a table of hard deleted projects just in case we need to reference them in something like the Moderation Log.

Reply:
I don't think so. Soft deleted projects will stay in the projects table and hard delete can only be done by admins and it should really remove all data then.